docs/guides/serverless.mdx

```diff
@@ -0,0 +1,40 @@
+---
+title: Introduction to Qiskit Serverless
+description: Use Qiskit Serverless to run quantum-classical workloads across the cloud and on quantum-centric supercomputers.
+---
+
+# What is Qiskit Serverless?
+
+Qiskit Serverless provides a simple interface to run workloads across quantum-classical resources. This includes deploying programs to the IBM Quantum&trade; Platform, running workloads remotely, and easy resource management for multi-cloud and quantum-centric supercomputing use cases.
+
+Premium users can build, deploy, and run their workloads remotely on classical compute made available through IBM Quantum Platform. Try out the tutorials in [IBM Quantum Learning](https://learning.quantum.ibm.com/catalog/tutorials?topics=qiskit-patterns) (note: these are accessible in the Premium Plan once you have logged into your IBM Quantum account).
+
+<Admonition type="note">
```

Comment:
For this experimental admonition could we use the same format that we use in other docs like here: https://docs.quantum.ibm.com/guides/qiskit-transpiler-service